DockerfileLeanFoundation

```diff
@@ -61,7 +61,7 @@ RUN conda clean -y --all
 
 # Install essencial packages
 RUN conda install -y                \
-    cython=0.29.15                  \
+    cython=0.29.17                  \
     numpy=1.18.1                    \
     pandas=0.25.3                   \
     wrapt=1.12.1
```

Comment:
Believe we could remove wrapt since it is no longer used. Python install readme specifies `pandas=0.23.4`